server/data/csv/b.csv

```diff
@@ -0,0 +1,4 @@
+0,Container for Contents,"Is it empty, rinsed and dried?"
+1,Food/drink Container,"Is it empty, rinsed and dried?"
+2,Normal Recyclable,-
+3,Paper Box,Flatten it
```

Comment:
@theodorayy actually I'm not sure about the usefulness of committing these data files since we have a copy in Google Drive - any reason why you'd want the data to be committed here?

Reply:
But for now I've followed this way and also added the csv file for `OneMapRecyclingBins`